CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,5 @@
 - Fix: NGSIv2 subscriptions payload validation (#1964, #1965, #1967, #1973, #1974, #1975, #1979, #1980, #1981, #1983, #1984, #1988, #2007, #2018, #2093, #2095, #2099, #2100)
+- Fix: Return error on v1 register requests with entity isPattern set to TRUE (Issue #2332)
```

Comment:
Alternatve wording suggestion:

Fix: Return error on **NGSIv1** register requests with entity isPattern set to **"true"** (Issue #2332)